### PR TITLE
[BUU] Fully enable admin_style_v3

### LINKED ISSUE #12823: [BUU] Fully enable admin_style_v3
⚠ Please use clockify code https://github.com/openfoodfoundation/openfoodnetwork/issues/7198 Back Office Uplift (Product) while reviewing or working on this task.

**TO BE RELEASED On September 3rd**
(see [schedule](https://community.openfoodnetwork.org/t/backoffice-ui-product-list-uplift-rollout/2921/35))

## Description
**- As a:** super admin
**- On page:** `/admin/feature-toggle/features`
**- I want to be able to do:**  See that on top of the admin and new users groups, the toggle is also activated for 100% of users

## Acceptance Criteria & Tests

1. Before the PR only admins + new user + 75% + dedicated actors eventually should have had access
2. After: feature is fully enabled

We will trust flipper to work on this one

### PR DESCRIPTION
**Pull request by chahmedejaz**
#### What? Why?

- Closes #12823
- This PR fully enables the `admin_style_v3` feature flag

#### What should we test?
- Disable the `admin_style_v3` feature flag if already enabled on the staging server
- Deploy this PR
- The `admin_style_v3` feature flag should be fully enabled after the successful deployment

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [x] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [ ] Technical changes only
- [ ] Feature toggled

#### Dependencies
 Wait until the following are merged: 
* https://github.com/openfoodfoundation/openfoodnetwork/pull/12826
* https://github.com/openfoodfoundation/openfoodnetwork/pull/12813

### CODE REVIEW

**Review by mkllnk — APPROVED** [review]
Just noting that we are waiting for other PRs to be merged before this one.

**Comment by dacook** [comment]
I've added one dependency to the description above, please add any others I'm not aware of.

**Review by dacook — APPROVED** [review]

**Comment by chahmedejaz** [comment]
I've updated the list to include one more bug fix which would be great to have before full rollout.

**Comment by RachL** [comment]
Given the last issue left is only on the Hungarian instance, I feel we can keep the current rollout process even if we don't manage to include it in this release (we are only moving from 75% to 100%, we are not removing the toggle). HU team has anyway disabled the feature meanwhile.

However if we do this, will the release process reactivate the feature for them?

**Comment by chahmedejaz** [comment]
Yes, Rachel. When the release is deployed to the Hungarian server then the feature toggle will be enabled by this issue. 😕

**Comment by chahmedejaz** [comment]
Yes, Rachel. When the release is deployed to the Hungarian server then the feature toggle will be enabled by this issue. 😕

**Comment by chahmedejaz** [comment]
The Hungarian issue has been fixed and approved, we just need one more approval to move it to testing.

**Pull request merged**
